src/containers/eventRSVP/index.tsx

```diff
@@ -0,0 +1,140 @@
+import { Alert, Space, Spin, Table, Typography } from 'antd';
+import React, { useEffect } from 'react';
+import { Helmet } from 'react-helmet';
+import { connect, useDispatch } from 'react-redux';
+import { Link as RouterLink, useParams } from 'react-router-dom';
+import { ChungusContentContainer } from '../../components';
+import { C4CState } from '../../store';
+import {
+  asyncRequestIsComplete,
+  asyncRequestIsFailed,
+  asyncRequestIsLoading,
+} from '../../utils/asyncRequest';
+import { getUpcomingEvents } from '../upcoming-events/ducks/thunks';
+import { EventsReducerState } from '../upcoming-events/ducks/types';
+import { getEventRegistrations } from './ducks/thunks';
+import { EventRegistrationsReducerState, Registration } from './ducks/types';
+
+const { Title, Link } = Typography;
+interface EventRSVPProps {
+  readonly events: EventsReducerState['upcomingEvents'];
+  readonly registrations: EventRegistrationsReducerState['eventRegistrations'];
+}
+
+const TablePhotoConsent = (text: Registration, record: Registration) => (
+  <Space size="middle">{record.photoRelease ? 'Yes' : 'No'}</Space>
+);
+const TableActions = (text: Registration, record: Registration) => (
+  <Space size="middle">
+    {/* TODO: Make the family details page and route to it here */}
+    <RouterLink to={`/family-details/${record.userId}`}>
+      <Link>View {record.firstName}'s Family Details</Link>
+    </RouterLink>
+  </Space>
+);
+
+const EventRSVP: React.FC<EventRSVPProps> = ({ events, registrations }) => {
+  const dispatch = useDispatch();
+  const id: number = parseInt(useParams<{ id: string }>().id, 10);
```

Comment:
what does the 2nd parameter `10` do

Reply:
Its the base of the number 😔 
I should really just use `Number()` instead.